languages/csharp/exercises/concept/operator-overloading/OperatorOverloadingTests.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using Xunit;
+
+public class OperatorOverloadingTests
+{
+    [Fact]
+    public void Equality_true()
+    {
+        Assert.True(Eq(new CurrencyAmount(55, "HD"), new CurrencyAmount(55, "HD")));
+    }
+
+    [Fact(Skip = "Remove this Skip property to run this test")]
+    public void Equality_false()
+    {
+        Assert.False(Eq(new CurrencyAmount(55, "HD"), new CurrencyAmount(60, "HD")));
+    }
+
+    [Fact(Skip = "Remove this Skip property to run this test")]
+    public void Equality_bad()
+    {
+        Assert.Throws<ArgumentException>(() => Eq(new CurrencyAmount(55, "HD"), new CurrencyAmount(60, "USD")));
+    }
+
+    private bool Eq(CurrencyAmount amountA, CurrencyAmount amountB)
```

Comment:
Maybe inline this method to make it clearer for the student what is happening?

Reply:
I think we should leave this to #2003.

My thinking was to encapsulate each operation in a separate method and to abstract away the reflection and maybe show the alternative non-reflective code under a conditional compilation directive but we can leave this up to the PR for #2003.